homeassistant/components/deconz/cover.py

```diff
@@ -38,19 +38,19 @@ async def async_setup_entry(
     gateway.entities[DOMAIN] = set()
 
     @callback
-    def async_add_cover(_: EventType, cover_id: str) -> None:
-        """Add cover from deCONZ."""
-        cover = gateway.api.lights.covers[cover_id]
-        async_add_entities([DeconzCover(cover, gateway)])
+    def async_add_light(_: EventType, light_id: str) -> None:
```

Comment:
Same here, I get that deCONZ handle covers as lights, but I don't think we should rename this all for that reason?

Reply:
I'm always back and forth on this thing... Regardless should be kept out of this pr. Thanks, I'll fix it right now

Reply:
Fixed